lib/ai/chat_test.go

```diff
@@ -51,7 +51,7 @@ func TestChat_PromptTokens(t *testing.T) {
 					Content: "Hello",
 				},
 			},
-			want: 610,
+			want: 1020,
```

Comment:
Probably not an issue for now, but these numbers are getting large for just "an opener".

Reply:
Yep. We might want to look into the function calling API to resolve this.